js/adapt-contrib-xapi.js

```diff
@@ -269,12 +271,30 @@ define([
       });
     },
 
+    /**
+     * Sends 'suspended' and 'terminated' statements to the LRS when the window
+     * is closed or the browser app is minimised on a device. Sends a 'resume'
+     * statement when switching back to a suspended session
+     */
+    onVisibilityChange: function() {
+      if (document.visibilityState === 'visible') {
+        return this.sendStatement(this.getCourseStatement(ADL.verbs.resumed));
+      }
+
+      if (!this.isComplete) {
+        // If the course is still in progress, send the 'suspended' verb.
+        this.sendStatementViaFetch(this.getCourseStatement(ADL.verbs.suspended));
+      }
+
+      // Always send the 'terminated' verb.
+      this.sendStatementViaFetch(this.getCourseStatement(ADL.verbs.terminated));
```

Comment:
Hi @danielstorey, apologies for the delay on this, but since the suspended and terminated statements are a special case in this instance, wouldn't it make sense to batch those in the `fetch` call?  

https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#212-post-statements indicates that a POST on /statements can take a single statement or an array of statements.  Something about the `keepalive: true` then dropping out of the function and running the same thing again doesn't seem right.